Source/CsDebugScript.Engine/DebugEventInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using CsDebugScript.Engine.Native;
+
+namespace CsDebugScript
+{
+    public class DebugEventInfo
+    {
+        /// <summary>
+        /// Event Type.
+        /// </summary>
+        public DEBUG_EVENT Type;
```

Comment:
These fields should be properties, probably something like:
public DEBUG_EVENT Type { get; internal set; }